app/controllers/home_controller.rb

```diff
@@ -14,7 +14,7 @@ def explore
   end
 
   def dashboard
-    @favorites = Workspace.accessible_by(current_ability, :update).favorites.order(updated_at: :desc).limit(3)
+    @favorites = current_user.favorite_workspaces.order(updated_at: :desc).limit(3)
```

Comment:
Should we be checking if still accessible here? It's possible for a user to fav a workspace they have access to and then lose access to the workspace. :man_shrugging: